### ISSUE TITLE
Uploading an existing project...

### ISSUE THREAD

**Issue by door43app** [unfoldingWord-dev/ts-desktop#883]
Notes
======
Uploading an existing project on door43 after making some changes

Environment
======
Environment Key | Value
:--: | :--:
Version |10.0.0
Operating System | Windows_NT
Platform | win32
Release | 10.0.14393
Architecture | x64

Log History
======
```javascript
2016-10-31 08:17:57 I/migrator.js:31:2: Migrated projects
	[]
2016-10-31 08:21:46 E/ts-dashboard.html:607:3: Restore Failed
	Error: ENOENT: no such file or directory, open 'C:\Users\tsqa1\AppData\Local\translationstudio\temp\en-x-demo1_rut_text_reg\manifest.json'
2016-10-31 08:22:2 E/ts-dashboard.html:607:3: Restore Failed
	Error: ENOENT: no such file or directory, open 'C:\Users\tsqa1\AppData\Local\translationstudio\temp\en-x-demo1_luk_text_reg\manifest.json'
2016-10-31 08:29:57 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_gen_text_reg",
	  "en-x-demo1_jud_text_reg",
	  "en-x-demo1_mat_text_reg",
	  "en-x-demo1_mrk_text_reg",
	  "en-x-demo1_obs_text_obs",
	  "en-x-demo1_phm_text_reg",
	  "en-x-demo1_php_text_reg",
	  "en-x-demo1_tit_text_reg",
	  "en-x-demo1_tit_tn",
	  "en-x-demo2_jud_text_reg",
	  "en-x-demo2_mat_text_reg"
	]
2016-10-31 08:37:11 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_gen_text_reg",
	  "en-x-demo1_jud_text_reg",
	  "en-x-demo1_mat_text_reg",
	  "en-x-demo1_mrk_text_reg",
	  "en-x-demo1_obs_text_obs",
	  "en-x-demo1_phm_text_reg",
	  "en-x-demo1_php_text_reg",
	  "en-x-demo1_tit_text_reg",
	  "en-x-demo1_tit_tn",
	  "en-x-demo2_jud_text_reg",
	  "en-x-demo2_mat_text_reg"
	]
2016-10-31 08:45:27 E/ts-dashboard.html:607:3: Restore Failed
	Error: ENOENT: no such file or directory, open 'C:\Users\tsqa1\AppData\Local\translationstudio\temp\en-x-demo1_rut_text_reg\manifest.json'
2016-10-31 08:45:39 E/ts-dashboard.html:607:3: Restore Failed
	Error: ENOENT: no such file or directory, open 'C:\Users\tsqa1\AppData\Local\translationstudio\temp\en-x-demo1_luk_text_reg\manifest.json'
2016-10-31 08:46:15 E/ts-dashboard.html:607:3: Restore Failed
	Error: ENOENT: no such file or directory, open 'C:\Users\tsqa1\AppData\Local\translationstudio\temp\en-x-demo1_luk_text_reg\manifest.json'
2016-10-31 08:56:22 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_gen_text_reg",
	  "en-x-demo1_gen_text_ulb",
	  "en-x-demo1_jud_text_reg",
	  "en-x-demo1_jud_text_ulb",
	  "en-x-demo1_mat_text_reg",
	  "en-x-demo1_mat_text_ulb",
	  "en-x-demo1_mrk_text_reg",
	  "en-x-demo1_mrk_text_ulb",
	  "en-x-demo1_obs_text_obs",
	  "en-x-demo1_phm_text_reg",
	  "en-x-demo1_phm_text_ulb",
	  "en-x-demo1_php_text_reg",
	  "en-x-demo1_php_text_ulb",
	  "en-x-demo1_tit_text_reg",
	  "en-x-demo1_tit_text_ulb",
	  "en-x-demo1_tit_tn",
	  "en-x-demo2_jud_text_ulb",
	  "en-x-demo2_mat_text_ulb",
	  "en-x-demo2_obs_text_obs",
	  "en-x-demo2_tit_tn",
	  "nl_obs_text_obs"
	]
2016-10-31 09:7:8 I/ts-dashboard.html:1072:3: 
	Git Version 2.9.2.windows.1
2016-10-31 09:7:32 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_gen_text_reg",
	  "en-x-demo1_gen_text_ulb",
	  "en-x-demo1_jud_text_reg",
	  "en-x-demo1_jud_text_ulb",
	  "en-x-demo1_mat_text_reg",
	  "en-x-demo1_mat_text_ulb",
	  "en-x-demo1_mrk_text_reg",
	  "en-x-demo1_mrk_text_ulb",
	  "en-x-demo1_obs_text_obs",
	  "en-x-demo1_phm_text_reg",
	  "en-x-demo1_phm_text_ulb",
	  "en-x-demo1_php_text_reg",
	  "en-x-demo1_php_text_ulb",
	  "en-x-demo1_tit_text_reg",
	  "en-x-demo1_tit_text_ulb",
	  "en-x-demo1_tit_tn",
	  "en-x-demo2_jud_text_ulb",
	  "en-x-demo2_mat_text_ulb",
	  "en-x-demo2_obs_text_obs",
	  "en-x-demo2_tit_tn",
	  "en_mat_text_reg",
	  "nl_obs_text_obs"
	]
2016-10-31 09:9:1 I/ts-dashboard.html:1072:3: 
	Git Version 2.9.2.windows.1
2016-10-31 09:9:24 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_gen_text_reg",
	  "en-x-demo1_gen_text_ulb",
	  "en-x-demo1_jud_text_reg",
	  "en-x-demo1_jud_text_ulb",
	  "en-x-demo1_mat_text_reg",
	  "en-x-demo1_mat_text_ulb",
	  "en-x-demo1_mrk_text_reg",
	  "en-x-demo1_mrk_text_ulb",
	  "en-x-demo1_obs_text_obs",
	  "en-x-demo1_phm_text_reg",
	  "en-x-demo1_phm_text_ulb",
	  "en-x-demo1_php_text_reg",
	  "en-x-demo1_php_text_ulb",
	  "en-x-demo1_tit_text_reg",
	  "en-x-demo1_tit_text_ulb",
	  "en-x-demo1_tit_tn",
	  "en-x-demo2_jud_text_ulb",
	  "en-x-demo2_mat_text_ulb",
	  "en-x-demo2_obs_text_obs",
	  "en-x-demo2_tit_tn",
	  "en_mat_text_reg",
	  "nl_obs_text_obs"
	]
2016-10-31 09:14:40 I/ts-dashboard.html:1072:3: 
	Git Version 2.9.2.windows.1
2016-10-31 09:14:52 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_gen_text_reg",
	  "en-x-demo1_jud_text_reg",
	  "en-x-demo1_mat_text_reg",
	  "en-x-demo1_mrk_text_reg",
	  "en-x-demo1_obs_text_obs",
	  "en-x-demo1_phm_text_reg",
	  "en-x-demo1_php_text_reg",
	  "en-x-demo1_tit_text_reg",
	  "en-x-demo1_tit_tn",
	  "en-x-demo2_jud_text_reg",
	  "en-x-demo2_mat_text_reg"
	]
2016-10-31 09:19:5 I/ts-dashboard.html:1072:3: 
	Git Version 2.9.2.windows.1
2016-10-31 09:19:5 I/migrator.js:31:2: Migrated projects
	[]
2016-10-31 09:19:39 I/ts-dashboard.html:1072:3: 
	Git Version 2.9.2.windows.1
2016-10-31 09:19:39 I/migrator.js:31:2: Migrated projects
	[]
2016-10-31 09:35:46 I/ts-dashboard.html:1072:3: 
	Git Version 2.9.2.windows.1
2016-10-31 09:36:9 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_gen_text_reg",
	  "en-x-demo1_gen_text_ulb",
	  "en-x-demo1_jud_text_reg",
	  "en-x-demo1_jud_text_ulb",
	  "en-x-demo1_mat_text_reg",
	  "en-x-demo1_mat_text_ulb",
	  "en-x-demo1_mrk_text_reg",
	  "en-x-demo1_mrk_text_ulb",
	  "en-x-demo1_obs_text_obs",
	  "en-x-demo1_phm_text_reg",
	  "en-x-demo1_phm_text_ulb",
	  "en-x-demo1_php_text_reg",
	  "en-x-demo1_php_text_ulb",
	  "en-x-demo1_tit_text_reg",
	  "en-x-demo1_tit_text_ulb",
	  "en-x-demo1_tit_tn",
	  "en-x-demo2_jud_text_ulb",
	  "en-x-demo2_mat_text_ulb",
	  "en-x-demo2_obs_text_obs",
	  "en-x-demo2_tit_tn",
	  "en_mat_text_reg",
	  "nl_obs_text_obs"
	]
2016-11-03 11:27:24 I/ts-dashboard.html:1072:3: 
	Git Version 2.9.2.windows.1
2016-11-03 11:27:48 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_gen_text_reg",
	  "en-x-demo1_gen_text_ulb",
	  "en-x-demo1_jud_text_reg",
	  "en-x-demo1_jud_text_ulb",
	  "en-x-demo1_mat_text_reg",
	  "en-x-demo1_mat_text_ulb",
	  "en-x-demo1_mrk_text_reg",
	  "en-x-demo1_mrk_text_ulb",
	  "en-x-demo1_obs_text_obs",
	  "en-x-demo1_phm_text_reg",
	  "en-x-demo1_phm_text_ulb",
	  "en-x-demo1_php_text_reg",
	  "en-x-demo1_php_text_ulb",
	  "en-x-demo1_tit_text_reg",
	  "en-x-demo1_tit_text_ulb",
	  "en-x-demo1_tit_tn",
	  "en-x-demo2_jud_text_ulb",
	  "en-x-demo2_mat_text_ulb",
	  "en-x-demo2_obs_text_obs",
	  "en-x-demo2_tit_tn",
	  "en_mat_text_reg",
	  "nl_obs_text_obs"
	]
2016-11-03 11:28:35 E/ts-dashboard.html:911:2: Upload Failed
	Error: getaddrinfo ENOENT git.door43.org:443
2016-11-03 11:29:45 E/ts-dashboard.html:911:2: Upload Failed
	Error: getaddrinfo ENOENT git.door43.org:443
2016-11-03 11:33:15 E/ts-dashboard.html:911:2: Upload Failed
	{
	  "stdout": "",
	  "stderr": "error: Ref refs/heads/master is at 09ab6104e846751e744c5551f91e6d2f9e7fe5fc but expected 0590077a8177c4febd3dce59673aee47919c9673\nremote: error: failed to lock refs/heads/master        \nTo git@git.door43.org:qa5/en-x-demo2_obs_text_obs.git\n ! [remote rejected] master -> master (failed to lock)\nerror: failed to push some refs to 'git@git.door43.org:qa5/en-x-demo2_obs_text_obs.git'\n",
	  "error": {
	    "killed": false,
	    "code": 1,
	    "signal": null,
	    "cmd": "cd \"C:\\Users\\tsqa1\\AppData\\Local\\translationstudio\\targetTranslations\\en-x-demo2_obs_text_obs\" && set GIT_SSH_COMMAND=ssh -i \"C:\\Users\\tsqa1\\AppData\\Local\\translationstudio\\ssh\\ts\" -o \"StrictHostKeyChecking no\" & git push -u git@git.door43.org:qa5/en-x-demo2_obs_text_obs.git master --follow-tags"
	  }
	}

```

**Comment by cckozie** [comment]
![image](https://cloud.githubusercontent.com/assets/8171759/19975606/950a4950-a1ba-11e6-9c46-fde0ea394855.png)

**Comment by cckozie** [comment]
I uploaded my OBS English Demo 2 project to door43. It indicated success, and showed me the link. When I clicked on the link and looked at the project on door43 it looked fine, but I noticed that the date on the project was pretty old. Assuming that was the case because I had uploaded it before and had not made any changes, I then went and added some text to story 2. Now when I try to upload it I get a nasty error message. (I did screen shots of the error msg, but too late noticed that snipet only stores the latest. I can rerun and capture the error message if that would be helpful.
